rfd/0081/health-check-draft.md

```diff
@@ -0,0 +1,58 @@
+# Mariposa Health Check Service
+
+The Mariposa (RFD36) health check service will be responsible for monitoring the health of user services. It establishes a set of supported health checks and provides an API for defining health checks that should be ran against project services. These health check definitions will be persisted in a proper data store (Postgres/Manatee?).
+
+The Health Check service process(es) will read user health check definitions, and coordinate which health check process will be responsible for running any given definition. 
+
+Failed health checks will be broadcast to interested services, such as the Convergence service.
+
+## Health check types
+
+The available health checks include:
+
+* HTTP(s) - Test the response of an HTTP request against a given service/port. Any part of the response may be validated including response code, headers, or body.
```

Comment:
> Any part of the response may be validated

We'll need to expand the definition in https://github.com/joyent/rfd/blob/master/rfd/0036/service-manifest.md#healthchecks to clarify how that's configured